.test-infra/jenkins/job_PostCommit_Java_ValidatesRunner_Dataflow_Java11.groovy

```diff
@@ -0,0 +1,41 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import CommonJobProperties as commonJobProperties
+import PostcommitJobBuilder
+
+
+PhraseTriggeringPostCommitBuilder.postCommitJob('beam_PostCommit_Java11_ValidatesRunner_Dataflow',
+  'Run Dataflow ValidatesRunner Java 11', 'Google Cloud Dataflow Runner ValidatesRunner Tests On Java 11', this) {
+
+  description('Runs the ValidatesRunner suite on the Dataflow runner with Java 11 worker harness.')
+
+  commonJobProperties.setTopLevelMainJobProperties(delegate, 'master', 180)
+
+  publishers {
+    archiveJunit('**/build/test-results/**/*.xml')
+  }
+
+  steps {
```

Comment:
In case of `job_PostCommit_Java_ValidatesRunner_Direct_Java11.groovy` we used 2 different `JAVA_HOME` env variables to run with 2 different java versions. Why is it not required here?

Reply:
The same as with [this PR](https://github.com/apache/beam/pull/8053), specifying the Java version is not required, since the tests run on Dataflow containers. @pabloem added selection of Java version on a Dataflow runner harness using the `java.specification.version` property, the image is pulled automatically with correct Java version based on this property.

Reply:
👍